File: Prak502/PRAK502-2210817210021-TRISNACAHYAPERMADI.c
#include <stdio.h>
#include <math.h>
int hitung (int nilai1, int nilai2){
    return nilai1 - nilai2;
}
int mutlak (int angka){
    return abs(angka);
}
int main (void)
{
    int a, b, c, d, Hasil;
    scanf("%i",&a);
    scanf("%i",&c);
    scanf("%i",&b);
    scanf("%i",&d);
    Hasil = hitung(a,b) + hitung(c,d);
    printf("%i",mutlak(Hasil));
    return 0;
}

File: Prak502/PRAK502-2210817210021-TRISNACAHYAPERMADI.py
def hitung (nilai1, nilai2) :
    return nilai1 - nilai2
def mutlak (angka) :
    return abs (angka)
a, c, b, d = input().split()
a=int(a)
b=int (b)
c=int(c)
d=int(d)
Hasil = hitung(a,b) + hitung(c,d)
print (mutlak(Hasil))

